error: Error Occured select * from "users" where "email" = $1 limit $2 - relation "users" does not exist {"code":"42P01","file":"parse_relation.c","length":104,"line":"1449","name":"error","position":"15","routine":"parserOpenTable","severity":"ERROR","stack":"error: select * from \"users\" where \"email\" = $1 limit $2 - relation \"users\" does not exist\n    at Parser.parseErrorMessage (/Users/suchaitgaurav/Documents/content-creation/auth-service/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/Users/suchaitgaurav/Documents/content-creation/auth-service/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/Users/suchaitgaurav/Documents/content-creation/auth-service/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/Users/suchaitgaurav/Documents/content-creation/auth-service/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:518:28)\n    at addChunk (node:internal/streams/readable:559:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)\n    at Readable.push (node:internal/streams/readable:390:5)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)"}
error: Error Occured secretOrPrivateKey must have a value {"stack":"Error: secretOrPrivateKey must have a value\n    at module.exports [as sign] (/Users/suchaitgaurav/Documents/content-creation/auth-service/node_modules/jsonwebtoken/sign.js:111:20)\n    at Users.create (/Users/suchaitgaurav/Documents/content-creation/auth-service/src/service/users.js:27:31)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async handleSignup (/Users/suchaitgaurav/Documents/content-creation/auth-service/src/controller/userController.js:17:27)"}
